### ISSUE TITLE
Anyway to delete a tag?

### ISSUE THREAD

**Issue by lavedon** [dvorka/mindforger#1409]
Hello, I'm loving Mind Forger.

Is there anyway to delete a tag? I can't seem to figure out how or find any relevant instructions in the documentation.

Thank you!

**Comment by dvorka** [comment]
Hello @lavedon!

Tags cannot be deleted - tags which you see listed (for instance when you adding tag to Note) are gathered from tags used in all your Notebooks and Notes. Tag disappears from that list if it is not used by any Notebook on Note (on MindForger restart). In order to find Notebooks or Notes which use particular tag you can use menu `Find` > Note or Notebook by tag.

Do you want me to implement an action which would delete particular tag from all Notebooks and Notes? (as a consequence tag would disappear also form all the lists)

**Comment by Iznik285** [comment]
From my perspective that would be a useful action to add

**Comment by Gulluth** [comment]
I am looking for a way to remove tags that I don't use. I believe they are built in tags. There are not any notebooks that contain the following tags, yet they are still present in the Tags screen and the Graph.

- obsolete
- later
- problem
- personal
- important
- cool
- done
- what
- how
- why
- where
- who

**Comment by djsudduth** [comment]
Agreed - I would also like to remove unused tags with some type of delete function. I'm not sure why those built-in tags are there and clutter up the knowledge graph.

**Comment by dvorka** [comment]
OK, makes sense - thank you for the enhancement suggestion!

Changing this GH issue from question to enhancement and adding it to the next release.
